takwimu/takwimu_ui/src/components/MakingOfTakwimu.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import { Typography } from '@material-ui/core';
+
+import Section from './Section';
+
+function MakingOfTakwimu() {
+  return (
+    <Section>
```

Comment:
Good idea on `Section` component. But make `title` a property so that it's consistent across all sections.